oracle-lib/src/cmr/oracle/connection.clj

```diff
@@ -14,6 +14,8 @@
    ;; This is required as an attempt to avoid NoClassDefFoundError occasionally during startup.
    ;; See CMR-3156
    (oracle.security.o5logon O5Logon)
+   ;;
```

Comment:
Add a comment if there's a reason to keep this commented out line otherwise remove.